Consumer/JobConsumer.php

```diff
@@ -41,13 +41,17 @@ public function execute(AMQPMessage $message)
             $output = $job->run($this->container);
 
             if (isset($data['uuid'])) {
-                $this->getJobLogRepository()->saveOutput(
-                    $data['uuid'],
-                    strval($output)
-                );
+                try {
+                    $this->getJobLogRepository()->saveOutput(
+                        $data['uuid'],
+                        strval($output)
+                    );
+                } catch (\Throwable $t) {
```

Comment:
Would that not be catched by parent `try/catch` ?

Reply:
Yeah but that’s causing the problem, the job completes fine but the saving of the job fails ... which isn’t a reason the flag the whole job as failed.